README.md

```diff
@@ -1,43 +1,65 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
 
-## Usage
+Baseline image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
 
-Follow these instructions if you are running docker *outside* of Google Compute Engine:
+Imaage includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
 
-    # Get the cloud sdk image:
-    $ docker pull google/cloud-sdk
 
-    # Auth & save the credentials in gcloud-config volumes:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud init
+## Dockerfile
```

Comment:
before showing how to build the image, you need 2 sections:

1. How do I use this image
  - how to pull
  - how to use
2. What are available tags, and links to their dockerfiles

e.g. 

![image](https://cloud.githubusercontent.com/assets/159209/26124991/5f9eb7ec-3a35-11e7-8a65-4d6260bfa58d.png)

I suggest trying as much as you can to make this look like it is one of the official docker hub images.